src/index.js

```diff
@@ -25,14 +25,15 @@ import CodeOfConduct from "./pages/Account/CodeOfConduct";
 
 import rootReducer from "./reducers/";
 import trackingMiddleware from "./middleware/tracking";
-import { API_KEY, APP_ID, MEASUREMENT_ID } from "./secrets";
+import { API_KEY, APP_ID, MEASUREMENT_ID, BUCKET_URL } from "./secrets";
```

Comment:
@thecadams You'll need this when deploying. Do you have a preferred way of sharing secrets / configs?